include/meta/meta_fwd.hpp

```diff
@@ -174,6 +174,24 @@
 #define META_IS_SAME(...) std::is_same<__VA_ARGS__>::value
 #endif
 
+#if defined(__GNUC__)
+#define META_IS_BASE_OF(...) __is_base_of(__VA_ARGS__)
+#elif META_CXX_TRAIT_VARIABLE_TEMPLATES
+#define META_IS_BASE_OF(...) std::is_base_of_v<__VA_ARGS__>
+#else
+#define META_IS_BASE_OF(...) std::is_base_of<__VA_ARGS__>::value
+#endif
+
+#if defined(__clang__)
+#define META_IS_CONSTRUCTIBLE(...) __is_constructible(__VA_ARGS__)
+#elif defined(__GNUC__) && __GNUC__ >= 8
+#define META_IS_CONSTRUCTIBLE(...) __is_constructible(__VA_ARGS__)
```

Comment:
This is identical to 186 - you could disjoin the conditions and eliminate the duplication. Also, MSVC supports this intrinsic.